charts/Makefile

```diff
@@ -0,0 +1,14 @@
+install-agent-prod:
+	@echo "Installing agent..."
+	helm install dd  --set datadog.apiKey=${DD_API_KEY} -f values.yaml datadog/datadog
+	@echo "Agent installed successfully."
+
+install-agent:
+	@echo "Installing agent..."
+	helm install dd -f values.yaml ./datadog  --set datadog.otelcollector.enabled=true --set-file datadog.otelcollector.config=config.yaml --set datadog.apiKey=${DD_API_KEY}
+	@echo "Agent installed successfully."
```

Comment:
after looking more at this file, I think you are using it for your local environment.
So it should not be commit.